### PR TITLE
Test cases for BZ #1836299 and #1873162

### PR DESCRIPTION
**Pull request by parikshithb**
Automates:  BZ #1836299 and #1873162

Signed-off-by: Parikshith <pbyregow@redhat.com>

### CODE REVIEW

**Review by fbalak — COMMENTED** [review]
Looks good. Just please move `test_hpa_noobaa_endpoint_metric` test case to `tests/manage/monitoring/prometheusmetrics` to separate alerting and metrics tests.

File: tests/manage/monitoring/prometheus/test_hpa.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    if len(alerts) > 0:
+        assert False, f"Failed: There should be no {constants.ALERT_KUBEHPAREPLICASMISMATCH} alert"
+
+
+@tier1
+@skipif_ocs_version('<4.6')
+@skipif_ocp_version('<4.6')
+@marks.polarion_id("OCS-2376")
+@marks.bugzilla('1873162')
+def test_hpa_noobaa_endpoint_metric():
```
Comment (mbukatov) [inline tests/manage/monitoring/prometheus/test_hpa.py:37]: I agree with Filip, I suggest to move this test function into `tests/manage/monitoring/prometheusmetrics/test_mcg_hpa.py` file.

**Review by mbukatov — COMMENTED** [review]
I would move `test_hpa_noobaa_endpoint_metric` test out of alerting directory, but otherwise it looks good.

**Review by mbukatov — DISMISSED** [review]

File: tests/manage/monitoring/prometheus/test_hpa.py (hunk 1 above)
Comment (parikshithb) [reply tests/manage/monitoring/prometheus/test_hpa.py]: ack

**Comment by parikshithb** [comment]
Both test cases verified.

File: tests/manage/monitoring/prometheusmetrics/test_mcg_hpa.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        kind=constants.HPA, resource_name='noobaa-endpoint',
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+
+    hpa = ocp_obj.get()['status']
+
+    if metric_key not in hpa:
+        assert False, "Failed: noobaa-endpoint cpu metrics is unavailable"
+    else:
+        assert hpa[metric_key] >= 0, "Failed: noobaa-endpoint cpu metrics is unknown"
+        logger.info(f"Current resource cpu utilized: {hpa[metric_key]}%")
```
Comment (ebenahar) [inline tests/manage/monitoring/prometheusmetrics/test_mcg_hpa.py:32]: Could be replaced with:
```python
assert metric_key in hpa, "Failed: noobaa-endpoint cpu metrics is unavailable"
assert hpa[metric_key] >= 0, "Failed: noobaa-endpoint cpu metrics is unknown"
logger.info(f"Current resource cpu utilized: {hpa[metric_key]}%")
```

**Review by fbalak — APPROVED** [review]

**Review by mbukatov — DISMISSED** [review]

File: tests/manage/monitoring/prometheusmetrics/test_mcg_hpa.py (hunk 2 above)
Comment (parikshithb) [reply tests/manage/monitoring/prometheusmetrics/test_mcg_hpa.py]: ack

**Review by PrasadDesala — APPROVED** [review]

**Review by ebenahar — DISMISSED** [review]

**Comment by parikshithb** [comment]
Had to force push again since Travis check was stuck for over 15 hours.

**Review by ebenahar — APPROVED** [review]

**Review by PrasadDesala — APPROVED** [review]

**Pull request merged**